libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationPerformanceEvent.kt

```diff
@@ -0,0 +1,54 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+import android.annotation.SuppressLint
+import android.os.Parcel
+import android.os.Parcelable
+import com.mapbox.android.telemetry.Event
+import java.text.SimpleDateFormat
+import java.util.ArrayList
+import java.util.Date
+import java.util.Locale
+
+@SuppressLint("ParcelCreator")
```

Comment:
`@SuppressWarnings("ParcelableCreator")` was removed, is it safe?

Reply:
It is not dangerous. Checked, lint is ok without this suppress.